### ISSUE TITLE
floccus  not sync favicon between client of Firefox 

### ISSUE THREAD

**Issue by tomergam** [floccusaddon/floccus#1115]
### Which version of floccus are you using?

Firefox 99.0.1 (64-bit) - Windows

### Sync method

{"label"=>"Nextcloud Bookmarks"}

### Which browser are you using?

Firefox 99.0.1 (64-bit)

### Which version of Nextcloud? (if relevant)

v23.0.2

### Describe the Bug

favicon is not sync between firefox client,
there is globe icon, only after opening the URL, the favicon  showing 

### Expected Behavior

the favicon should appear in you environment  after full sync

### To Reproduce

Installed new clean browser with floccus extension 

### Debug log provided

- [ ] I have provided a debug log file

**Comment by cracket** [comment]
I will have to counter this bug, because I have observed something opposite.
I have pulled my bookmarks from webdav using floccus 4.17.1 on firefox esr 102.3.0 fresh installation. 
Synchronization of thousands of items took 15 minutes. Half hour after synchronization machine was still experiencing heavy network traffic without any browser activity. I was shocked because FF was downloading every single favicon from the internet for each new bookmark that was pulled, and after another half hour every "globe" icon was replaced with original one. 
So after some time firefox did synced favicons on it's own extremely slowing down my network connectivity. 

This is obviously not a bug of floccus as far as I can see, but limitation of xbel. But if you discuss it - make sure you take into account privacy&performance impact related to it:
- privacy: fetching favicons can leak correlation between pages browsed and bookmarked using different ip addresses, browsers and include private mode of single floccus user
- performance: floccus should not be cascading extra load on machine&network

I recommend also reading a bug report from mozilla about favicons and impact of caching them:
https://bugzilla.mozilla.org/show_bug.cgi?id=1618257
which suggests that even if floccus synced favicons, ff would ignore the cache and reload favicons directly

There is also nice document mentioned there about favicon based fingerprinting
https://www.cs.uic.edu/~polakis/papers/solomos-ndss21.pdf

So I suggest keeping this issue "On hold" and participating in discussion about firefox behavior on favicons.
